source/com.google.firebase/firebase-installations-interop/source/Firebase-Iid.csproj

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we no longer generate these project files?

Reply:
We. Do. Not sure what happened here. Removed